### DISCUSSION TITLE
How can I change the current working directory of exe?

### DISCUSSION THREAD

**Discussion by DFVSQY** [microsoft/vscode-cmake-tools#2824] [Q&A] [answered]
I have a project folder `D:\LearnGameRepo\Lua`, and I output the executable file to `D:/LearnGameRepo/Lua/build/bin/main_mymath.exe`, then I click launch icon in the status bar and the current working directory of the executable file is `D:\LearnGameRepo\Lua\build\bin`:
![image](https://user-images.githubusercontent.com/8075421/198173943-94c0c2d7-eb44-47ff-a77f-5c738b73c3d5.png)

I want to know how can i change the current working directory of exe to `D:\LearnGameRepo\Lua`?
Please give me some help, Thank you very much.

**Comment by bobbrow** [comment, marked as answer]
You can use this setting to change the working directory:

```json
  "cmake.debugConfig": {
    "cwd": "D:\\LearnGameRepo\\Lua"
  }
```
